packages/twenty-front/src/modules/ui/input/color-scheme/components/ColorSchemePicker.tsx

```diff
@@ -35,31 +35,37 @@ export const ColorSchemePicker = ({
   value,
   onChange,
   className,
-}: ColorSchemePickerProps) => (
-  <StyledContainer className={className}>
-    <StyledCardContainer>
-      <ColorSchemeCard
-        onClick={() => onChange('Light')}
-        variant="Light"
-        selected={value === 'Light'}
-      />
-      <StyledLabel>Light</StyledLabel>
-    </StyledCardContainer>
-    <StyledCardContainer>
-      <ColorSchemeCard
-        onClick={() => onChange('Dark')}
-        variant="Dark"
-        selected={value === 'Dark'}
-      />
-      <StyledLabel>Dark</StyledLabel>
-    </StyledCardContainer>
-    <StyledCardContainer>
-      <ColorSchemeCard
-        onClick={() => onChange('System')}
-        variant="System"
-        selected={value === 'System'}
-      />
-      <StyledLabel>System settings</StyledLabel>
-    </StyledCardContainer>
-  </StyledContainer>
-);
+}: ColorSchemePickerProps) => {
+  const handleOnChange = (ColorScheme: ColorScheme) => {
+    localStorage.setItem('app-theme', ColorScheme);
```

Comment:
Not a fan of introducing this, I think there are more elegant ways to solve this :)
Would you mind reverting the changes related to that? We can always solve them in another PR/issue and take time to do it properly (sorry I know this kind of contradict what was said before)

Reply:
Got it!
I just wanted to point out the theme inconsistencies.
Will revert it.
And yes this could be solved in much more elegant way, I was trying to figure out how this works.
Thanks